### PR TITLE
Changed base image to audited Debian Base

### PR DESCRIPTION
**Pull request by davidz627**
Fixes: #15 

/assign @msau42

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,9 +15,13 @@
...
 
-FROM alpine:3.7
+# Start from Google Debian base
+FROM gcr.io/google-containers/debian-base-amd64:0.3
 COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/bin/gce-pd-csi-driver /gce-pd-csi-driver
 
-ENTRYPOINT ["/gce-pd-csi-driver"]
\ No newline at end of file
+# Install necessary dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends util-linux e2fsprogs mount ca-certificates
```
[the remark this answers is not in the document]
Comment (davidz627) [reply Dockerfile]: fixed

**Comment by davidz627** [comment]
addressed comments

File: Dockerfile [hunk 2]
```diff
@@ -15,9 +15,13 @@
...
 
-FROM alpine:3.7
+# Start from Google Debian base
+FROM gcr.io/google-containers/debian-base-amd64:0.3
 COPY --from=builder /go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/bin/gce-pd-csi-driver /gce-pd-csi-driver
 
-ENTRYPOINT ["/gce-pd-csi-driver"]
\ No newline at end of file
+# Install necessary dependencies
+RUN clean-install -y --no-install-recommends util-linux e2fsprogs mount ca-certificates
```
Comment (msau42) [inline Dockerfile]: you don't need the options, clean-install already runs with them
https://github.com/kubernetes/kubernetes/blob/master/build/debian-base/clean-install

**Comment by davidz627** [comment]
comments addressed

**Pull request merged**